charts/internal/shoot-storageclasses/templates/storageclasses.yaml

```
@@ -22,4 +22,14 @@ provisioner: pd.csi.storage.gke.io
 parameters:
   type: pd-ssd
 volumeBindingMode: WaitForFirstConsumer
+
+---
+apiVersion: snapshot.storage.k8s.io/v1beta1
+kind: VolumeSnapshotClass
+metadata:
+  annotations:
+    snapshot.storage.kubernetes.io/is-default-class: "true"
+  name: gardener-default
```

See https://github.com/gardener/gardener-extension-provider-aws/pull/146#discussion_r459248980
```suggestion
  name: default
```

---

done